homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: todo
+  config-flow: done
+  config-flow-test-coverage: todo
+  dependency-transparency: done
+  docs-actions: todo
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: todo
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: todo
+  test-before-setup: todo
+  unique-config-entry: todo
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: done
+  integration-owner: done
```

Comment:
There are currently 2 owners https://github.com/home-assistant/core/blob/dev/homeassistant/components/awair/manifest.json#L4

If needed, I'm happy to be included as well

Reply:
You can take my spot. While I have been involved before, I'm not active anymore.